lib/sandthorn_sequel_projection.rb

```diff
@@ -22,7 +21,7 @@ class << self
     require 'delegate'
     extend Forwardable
 
-    def_delegators :configuration, :batch_size, :event_stores
+    def_delegators :configuration, :batch_size, :event_store
```

Comment:
We could change it back to `event_stores` and set it to a Hash `{deafult: deafult_driver, other_driver: some_other_driver}` like in Sandthorn